website/blog/releases/v1.0.1.md

```diff
@@ -0,0 +1,99 @@
+---
+date: 2025-01-27
+title: 'Spice v1.0.1 (Jan 27, 2025)'
+type: blog
+authors: [sevenan]
+tags: [aws, iceberg, release, python, sdk, gpu]
```

Comment:
```suggestion
tags: [aws, iceberg, release, python, sdk, gpu, cuda, metal, cli]
```

@karifabri let's also add tags for cuda and metal, and ensure they are in tags.yml